Core/Versioning/KspVersionCriteria.cs

```diff
@@ -42,6 +42,21 @@ public KspVersionCriteria Union(KspVersionCriteria other)
             );
         }
 
+        public override bool Equals(object obj)
```

Comment:
One more thing I just spotted:

> It would be less bad if this was an unusual special case, but any class that overrides Equals is going to get a compiler warning requesting an implementation of GetHashCode

>   KspVersionCriteria.cs(7, 18): [CS0659] 'KspVersionCriteria' overrides Object.Equals(object o) but does not override Object.GetHashCode()

:grin:

Reply:
Added.